setup.sh

```diff
@@ -15,6 +15,20 @@ sudo apt-get install -y postgresql postgresql-contrib
 echo "install requirements"
 pip3 install -r /vagrant/requirements.txt
 
+echo "installing postgresql"
+sudo apt-get install -y postgresql postgresql-contrib
+
+echo "configuring database"
+sudo su postgres <<POSTGRESQL_COMMANDS
+psql
+CREATE DATABASE edison;
+ALTER ROLE postgres WITH PASSWORD 'edison';
+POSTGRESQL_COMMANDS
+
 echo "running flask_init.py"
 export FLASK_APP=/vagrant/flask_init.py
-python3 -m flask run --host=0.0.0.0 >> /vagrant/log.log 2>&1 &
+flask run -h 0.0.0.0 -p 5000 >> /vagrant/flask_init.log 2>&1 &
+
+echo "running app.py"
+export FLASK_APP=/vagrant/backend/app.py
+flask run -h 0.0.0.0 -p 3000 >> /vagrant/app.log 2>&1 &
```

Comment:
I don't understand why flask is executed twice

Reply:
The first flask is the frontend server that will send static files to the client.
The second flask is the Restful API server which contains the business logic of our app.

Reply:
ack